docs/components/modeler/about.md

```diff
@@ -11,20 +11,14 @@ In tandem, different events and implementation details (such as the conditions w
 
 Camunda offers a few tools to design your diagrams and implement them:
 
-- [Cloud Modeler](./cloud-modeler/launch-cloud-modeler.md)
-- [Camunda Modeler](./camunda-modeler/install-the-modeler.md)
+- [Web Modeler](./web-modeler/launch-cloud-modeler.md)
+- [Desktop Modeler](./desktop-modeler/install-the-modeler.md)
 
-:::note
-You can also utilize [Cawemo](https://cawemo.com/), which focuses on the design phase and provides options for collaborative work on a BPMN model.
-:::
+Web Modeler and Desktop Modeler differ mainly in their environment. Web Modeler is part of Cloud Console and offers a seamless integration into Camunda Cloud to model BPMN. Desktop Modeler is a desktop application that can be installed and used locally, all while integrating your local development environment.
```

Comment:
Should we expain that Web Modeler is compatabile with C8 and Desktop Modeler is compatabile with C7 and C8? Otherwise people might try to use Web Modeler for C7 and be frustrated when learning that this won't work?

@MoAtCamunda

Reply:
Hi Max, How would customers be able to use web modeler with C7? In SaaS Web Modeler, I dont believe theres a way for customers to leverage C7. For Self managed Web Modeler that makes sense but we on dont have that available today. 

We could explicitly state desktop is for C7 & C8 if that makes sense

Reply:
Yes - that is what I meant. Let's state desktop is for C7 and C8 and web is for C8.

Reply:
Aa reminder, this is a public repo. I'm going to go ahead and delete these comments, but take a screenshot and move the conversation internally.